plugin.xml

```diff
@@ -4,6 +4,7 @@
   id="cordova-plugin-image-picker"
   version="1.1.1">
 
+	<dependency id="cordova-plugin-donkeyfont" url="https://github.com/zenput/cordova-plugin-donkeyfont" commit="master" />
```

Comment:
Once everything's good to go, we should make a release tag on the donkeyfont repo and point this to that instead of just the `master` branch. Less dangerous.